layers/meta-balena-compulab/recipes-kernel/linux/linux-imx_4.14.%.bbappend

```diff
@@ -13,12 +13,15 @@ SRC_URI_append = " \
 
 SRC_URI_append_etcher-pro = " \
 	file://0001-linux-imx-Enable-I2C4-in-cl-som-imx8-dtb.patch \
+	file://0001-Enable-the-two-PCA9956B-LED-drivers-in-dtb.patch \
 "
 
 
 KERNEL_IMAGETYPE_cl-som-imx8 = "Image.gz"
 
 RESIN_CONFIGS_append = " 80211 iwlwifi overlayfs debug_preempt_disable"
+RESIN_CONFIGS_append_etcher-pro = " pca9956b"
```

Comment:
Please move this right next to `RESIN_CONFIGS[pca9956b] = ` so it is more visible